packages/game-bridge/src/index.ts

```diff
@@ -231,6 +232,7 @@ window.callFunction = async (jsonData: string) => {
             redirectUri: redirect ?? redirectUri,
             logoutRedirectUri: request?.logoutRedirectUri,
             crossSdkBridgeEnabled: true,
+            logoutMode,
```

Comment:
If the logout mode is `silent` for device code auth, how does it clear the session from the browser without opening it?

Reply:
Looks like if paired with HardLogout from the Game SDKs, that should force a Passport login to happen.